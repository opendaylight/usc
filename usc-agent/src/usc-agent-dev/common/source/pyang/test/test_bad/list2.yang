module list2 {
  namespace "urn:test:list2";
  prefix "l";

  choice test {
    default node;
    case node {
      leaf  mandatory {
        type string;
        mandatory true; // error: no mandatory under default case
      }
      list mandatory2 { // error: no mandatory under default case
        min-elements "3";
        key "name";
        leaf name {
          type string;
        }
      }
      container x {
        container y {
          leaf-list m {
            min-elements "1"; // error: no mandatory under default case
            type string;
          }
        }
      }
    }
  }
  
} 
