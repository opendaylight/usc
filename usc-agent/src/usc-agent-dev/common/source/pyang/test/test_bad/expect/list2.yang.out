list2.yang:8: error: MANDATORY_NODE_IN_DEFAULT_CASE
list2.yang:12: error: MANDATORY_NODE_IN_DEFAULT_CASE
list2.yang:21: error: MANDATORY_NODE_IN_DEFAULT_CASE
